https/README.md

```diff
@@ -0,0 +1,24 @@
+# HTTPS Package for Prometheus
+
+The `https` directory contains a Go package and a sample configuration file for running `node_exporter` with HTTPS instead of HTTP.
+When running a server with TLS use the flag `--web.authentication-config`
```

Comment:
since eventually the file will hold  configuration for both TLS and authentication, maybe we should name the flag `--web.config`?